usagov/vote-gov-drupal
# Vote 898: style homepage components

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[VOTE: 898](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-898&assignee=609161faf6c0960069bf61b3)

## Description (optional)
This PR handles the styling of homepage components:
- Registration Tool 
- Accordion Group 
- Basic Block (Already registered)

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. Pull branch
2. Import db
3. Run lando retune

### Testing steps
1. Create content for each homepage block
4. Confirm each block matches the style of current Vote.gov site 1:1
